src/IW/Sync/Update.hs

```diff
@@ -0,0 +1,25 @@
+module IW.Sync.Update where
+
+import Control.Monad.IO.Unlift (MonadUnliftIO)
+import UnliftIO.Async (mapConcurrently_)
+
+import IW.App (WithError)
+import IW.Db
+import IW.Effects.Cabal (MonadCabal (..))
+import IW.Sync.Search
+
+
+fetchAndUpsertRepos
+    ::
+    ( MonadCabal m
+    , MonadUnliftIO m
+    , WithDb env m
+    , WithLog env m
+    , WithError m
+    )
+    => m ()
+fetchAndUpsertRepos = do
+    gitHubRepos <- fetchAllHaskellRepos
+    let repos = fromGitHubRepo <$> gitHubRepos
```

Comment:
I usually prefer to use `map` when working with ordinary lists. Sometimes using monomorphic functions over polymorphic ones have readability benefits 🙂

Reply:
@chshersh That's true :+1: